_articles/legal.md

```diff
@@ -12,17 +12,17 @@ related:
 
 ## Understanding the legal implications of open source
 
-Sharing your creative work with the world can be an exciting and rewarding experience. It can also mean a bunch of legal things you didn't know you had to worry about. Thankfully, you don't have to start from scratch. We've got your legal needs covered. (Before you dig in, be sure to read our [disclaimer](/notices/).)
```

Comment:
This article does not and cannot "cover the reader's legal needs". Rather it is a broad introduction to the legal side of OSS.